gravitee-apim-console-webui/src/management/management.module.ajs.ts

```diff
@@ -547,19 +546,15 @@ require('diff/dist/diff.min.js');
 require('angular-loading-bar');
 
 // Highcharts
-import * as Highcharts from 'highcharts';
-(<any>window).Highcharts = Highcharts;
-import HighchartsMore from 'highcharts/highcharts-more';
-import SolidGauge from 'highcharts/modules/solid-gauge';
-import NoDataToDisplay from 'highcharts/modules/no-data-to-display';
-import Map from 'highcharts/modules/map';
-
-HighchartsMore(Highcharts);
-SolidGauge(Highcharts);
-NoDataToDisplay(Highcharts);
-Map(Highcharts);
-
-import '@highcharts/map-collection/custom/world';
+const Highcharts = require('highcharts/highstock');
```

Comment:
Is should be `highcharts` and not `highcharts/highstock` no?

Reply:
I don't know, we can try. I found this solution in an issue of highcharts about es6

Reply:
:youhoo: it's work. 🚀